app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt

```diff
@@ -32,27 +32,42 @@ class AdministratorControlsViewModel @Inject constructor(
   private lateinit var userProfileId: ProfileId
 
   private val deviceSettingsLiveData: LiveData<DeviceSettings> by lazy {
-    Transformations.map(profileManagementController.getDeviceSettings(), ::processGetDeviceSettingsResult)
+    Transformations.map(
+      profileManagementController.getDeviceSettings(),
+      ::processGetDeviceSettingsResult
+    )
   }
 
   val administratorControlsLiveData: LiveData<List<AdministratorControlsItemViewModel>> by lazy {
     Transformations.map(deviceSettingsLiveData, ::processAdministratorControlsList)
   }
 
-  private fun processGetDeviceSettingsResult(deviceSettingsResult: AsyncResult<DeviceSettings>): DeviceSettings {
+  private fun processGetDeviceSettingsResult(
+    deviceSettingsResult: AsyncResult<DeviceSettings>
+  ): DeviceSettings {
     if (deviceSettingsResult.isFailure()) {
-      logger.e("AdministratorControlsFragment", "Failed to retrieve profile", deviceSettingsResult.getErrorOrNull()!!)
+      logger.e(
+        "AdministratorControlsFragment",
+        "Failed to retrieve profile",
+        deviceSettingsResult.getErrorOrNull()!!
+      )
     }
     return deviceSettingsResult.getOrDefault(DeviceSettings.getDefaultInstance())
   }
 
-  private fun processAdministratorControlsList(deviceSettings: DeviceSettings): List<AdministratorControlsItemViewModel> {
+  private fun processAdministratorControlsList(
+    deviceSettings: DeviceSettings
+  ): List<AdministratorControlsItemViewModel> {
     val itemViewModelList: MutableList<AdministratorControlsItemViewModel> = mutableListOf(
       AdministratorControlsGeneralViewModel()
     )
     itemViewModelList.add(AdministratorControlsProfileViewModel(routeToProfileListListener))
-    itemViewModelList.add(AdministratorControlsDownloadPermissionsViewModel(fragment, logger, profileManagementController,
-      userProfileId, deviceSettings))
+    itemViewModelList.add(
+      AdministratorControlsDownloadPermissionsViewModel(
+        fragment, logger, profileManagementController,
+        userProfileId, deviceSettings
```

Comment:
Is this correct or should this be in multiple lines like:
```
        fragment,
        logger,
        profileManagementController,
        userProfileId,
        deviceSettings
```

Reply:
both ways are correct.

Reply:
Okay, then in that case, keep it in multiple lines for better code clarity and consistency with https://github.com/oppia/oppia-android/pull/1291#discussion_r439205779

Reply:
ok